livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/realtime/realtime_model.py

```diff
@@ -1662,13 +1685,12 @@ async def _run_fnc_task(self, fnc_call_info: llm.FunctionCallInfo, item_id: str)
                 "function": fnc_call_info.function_info.name,
             },
         )
-        if called_fnc.result is not None:
-            create_fut = self.conversation.item.create(
-                tool_call,
-                previous_item_id=item_id,
-            )
-            await self.response.create(on_duplicate="keep_both")
-            await create_fut
+        create_fut = self.conversation.item.create(
+            tool_call,
+            previous_item_id=item_id,
+        )
+        await self.response.create(on_duplicate="keep_both")
+        await create_fut
```

Comment:
any reason to change this logic, this is actually an expected thing where we don't want to send results to the LLM if the aifunction returns None

Reply:
When Function returns exception, `called_fnc.result` is `None`. and we are adding exception in content. maybe we should check `if tool_call.content is not None:` I'll update it

Reply:
I don't think we want to do that, if there is an exception, the LLM should know

Reply:
Yeah we will send it through content, like we do in pipeline

Reply:
ah ok makes sense